piptools/utils.py

```diff
@@ -628,7 +633,6 @@ def get_click_dest_for_option(option_name: str) -> str:
 ]
 
 
-@functools.lru_cache()
```

Comment:
If this is for testing, the resulting callable contains a `__wrapped__` attribute and you could make use of it with `monkeypatch.setattr` in tests instead of dropping the cache in normal runtime.

Reply:
The cache causes tests to become flaky and fail, depending on their ordering. I wonder why this is cached in the first place, especially as this function is only called once.

Reply:
Okay, fair.

Reply:
Marking this as resolved.